brainstorm/src/components/group.tsx

```diff
@@ -124,35 +130,6 @@ export function GroupView(props: {
 		return unsubscribe;
 	}, []);
 
-	const [parent, setParent] = useState(Tree.parent(props.group));
```

Comment:
parent was only used in drag and drop callbacks, so to avoid having to invalidate it, it is now recomputed when needed.